File: lesson1/buttonclicker/src/main/java/ru/mirea/chmykhovam/buttonclicker/MainActivity.java
package ru.mirea.chmykhovam.buttonclicker;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.CheckBox;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    private TextView textViewStudent;
    private Button btnWhoAmI;
    private Button btnItIsNotMe ;
    private CheckBox checkBox;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        textViewStudent = findViewById(R.id.tvOut);
        btnWhoAmI = findViewById(R.id.btnWhoAmI);
        btnItIsNotMe = findViewById(R.id.btnItIsNotMe);
        checkBox = findViewById(R.id.checkBox);
        View.OnClickListener oclBtnWhoAmI = new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                textViewStudent.setText(R.string.myNumText);
                checkBox.toggle();
            }
        };
        btnWhoAmI.setOnClickListener(oclBtnWhoAmI);
    }
    public void onClickItIsNotMe(View v){
        textViewStudent.setText(R.string.itIsNotMeText);
        checkBox.toggle();
    }
}

File: Lesson2/toastapp/src/main/java/ru/mirea/chmykhovam/toastapp/MainActivity.kt
package ru.mirea.chmykhovam.toastapp

import android.os.Bundle
import android.view.View
import android.widget.EditText
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun onClickMakeToast(view: View?){
        val editTextLen = findViewById<EditText>(R.id.editText).text.length
        val toast = Toast.makeText(
            applicationContext,
            "СТУДЕНТ № 31 ГРУППА БСБО-05-21 Количество символов - $editTextLen",
            Toast.LENGTH_SHORT
        )
        toast.show()
    }
}